zug/transducer/map_indexed.hpp

```diff
@@ -19,7 +20,7 @@ namespace zug {
 template <typename MappingT>
 constexpr auto map_indexed(MappingT&& mapping)
 {
-    return comp(count(), map(std::forward<MappingT>(mapping)));
+    return detail::make_transducer_holder(comp(count(), map(std::forward<MappingT>(mapping))));
```

Comment:
Ignore these changes... The changes to the composed transducers won't be needed.